src/components/Notification/StatusMessage.jsx

```diff
@@ -18,19 +24,35 @@ import React from 'react';
 const StatusMessage = ({ notification, locationUrl }) => {
   if (locationUrl) {
     return (
-      <dd className="labelStatus" role="alert">
-        {notification}{' '}
-        <a href={locationUrl} target="_blank" rel="noreferrer">
-          {locationUrl}
+      <Typography
+        variant="dd"
+        sx={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center'
+        }}
```

Comment:
Lines 29 to 33 can be removed. Styling for centering the text is already handled by StatusNotification via `<Box>`.